frontend/uno.config.ts

```diff
@@ -34,5 +41,28 @@ export default defineConfig({
         "components/ui/**/*.{js,ts}"
       ]
     }
-  }
+  },
+  safelist: [
```

Comment:
Why! Maybe improve `content.pipeline.include` to search for for files in components instead of only `components/ui`